Reference_manual.py

```diff
@@ -0,0 +1,2 @@
+def color_pair_to_string(major_color, minor_color):
```

Comment:
The "New feature request" in the README.md is about printing all the numbers and color pairs. Pls add that feature

Reply:
Ok Sudeep, i'll do that.